src/qt/forms/importkeysdialog.ui

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>ImportKeysDialog</class>
+ <widget class="QDialog" name="ImportKeysDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>560</width>
+    <height>440</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Import Private Key</string>
+  </property>
+  <property name="modal">
+   <bool>true</bool>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QTabWidget" name="tabWidget">
+     <property name="currentIndex">
+      <number>0</number>
+     </property>
```

Comment:
I'd remove these, there's no need for the tab widget as there is only one view. Result:
<img width="558" alt="Screen Shot 2021-05-24 at 02 11 27" src="https://user-images.githubusercontent.com/1410115/119281520-4eb9fe00-bc36-11eb-8c9a-ac68a9fd7f7e.png">